src/language-css/printer-postcss.js

```diff
@@ -409,6 +410,12 @@ function genericPrint(path, options, print) {
         const iNextNode = node.groups[i + 1];
         const iNextNextNode = node.groups[i + 2];
 
+        if (
+          iNode.type === "value-func" &&
+          hasNoSpaceBeforeValueNode(iNextNode)
```

Comment:
Better add helper `isPostcssSimpleVar` and move all check inside this util function.